### DISCUSSION TITLE
Patterns wishlist

### DISCUSSION THREAD

**Discussion by jdanyow** [microsoft/atlas-design#68] [General]
Do you have a pattern you'd like to see documented? Add it as a top-level comment below! Upvote the ideas you find useful 👍

**Comment by jdanyow** [comment]
New tab links
- When to open in a new tab by default
- What a link should look like / convey to assistive tech when it will open in a new tab by default
- Differentiate from external links
- Current scenarios on Docs and ACOM
    - todo...

**Comment by wibjorn** [comment]
I would like to see us implement a reading-width that uses clamp under the hood.

https://jsfiddle.net/wibjorn/z7ej4k6d/

**Reply by 5t3ph** [reply]
Definitely +1 on this idea!

If I may suggest a slight update that gets the same result but without the unused explicit minimum: `min: (50ch, 100%)`

it gets the "minimum" at any given time which results in the 100% being selected on smaller viewports/in smaller containers and 50ch selected on large viewports/in larger containers. And it's [the same browser support as clamp](https://caniuse.com/css-math-functions).

**Reply by Courtny** [reply]
Yes please.

**Comment by wibjorn** [comment]
The [stretched link](https://docsstyleguide.z13.web.core.windows.net/stretched-link.html) pattern
